### PR TITLE
Add burn functionality

### PR DESCRIPTION
**Pull request by mariavarvaroi**

### CODE REVIEW

**Comment by mariavarvaroi** [comment]
Gas estimate seems off:
Screenshot 2024-02-08 at 11 37 09

File: book/guide/8-burn-os-token.md [hunk 1]
```diff
@@ -1,8 +1,51 @@
 ## 8. Burn OS Token
 
-In this section, we will learn about the burning process for the Opus pool SDK. The burning process is used to destroy the minted shares and receive the underlying assets back. This process is required when the user wants to unstake the whole staked amount. You can learn more about staking [here][stake] and unstaking [here][unstake].
+In this section, we will learn about the burning process for the Opus pool SDK. The burning process is used to destroy the minted shares and receive the underlying assets back. This process is required if the user minted a portion or the entire staked amount and now wants to unstake. You can learn more about staking [here][stake] and unstaking [here][unstake].
 
-    This section is under development
+The stes to burn OS token are as follows:
```
Comment (ayushrungta8) [inline book/guide/8-burn-os-token.md:5]: minor typo I guess `stes`

Comment (mariavarvaroi) [reply book/guide/8-burn-os-token.md]: yup, saw that, fixed!

File: book/guide/8-burn-os-token.md [hunk 2]
```diff
@@ -1,8 +1,51 @@
...
+    type: 'eip1559',
+    gas: burnTx.gasEstimation,
+    maxPriorityFeePerGas: burnTx.maxPriorityFeePerGas,
+    maxFeePerGas: burnTx.maxFeePerGas,
+});
+```
+
+## Next Steps
+
+In this section, we focused on the core functionality of our application, which is submitting a burning transaction. To continue exploring the functionality of our application, you can proceed to the next section: [Unstaking Functionality][unstake].
```
Comment (welldan97) [inline book/guide/8-burn-os-token.md:48]: That's a bit confusing.  if unstaking is the next page, then the order should be different. unstake page should not be page #4, it should be going after burn. Would be good idea to check if the flow still holds

Comment (mariavarvaroi) [reply book/guide/8-burn-os-token.md]: fixed [ee02192](https://github.com/ChorusOne/opus-pool-sdk/pull/12/commits/ee02192ef41595da3f56ae9039b796e277ddb17b)

**Review by welldan97 — APPROVED** [review]
Great! Thank you

**Pull request merged**